scripts/ops/safeAllocation.ts

```diff
@@ -0,0 +1,97 @@
+import fs from "fs-extra";
+import { ethers } from "hardhat";
+import Safe from "@gnosis.pm/safe-core-sdk";
+import EthersAdapter from "@gnosis.pm/safe-ethers-lib";
+import { SafeEthersSigner, SafeService } from "@gnosis.pm/safe-ethers-adapters";
+import { Wallet } from "ethers";
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { chefV2Address, ops } from "../constants";
+
+type AllocationConfig = {
+  PoolId: number;
+  Allocation: number;
+  Rewarder: string;
+  LpToken: string;
+};
+
+const { SAFE_SIGNER_MNEMONIC = undefined, AURORA_API_KEY } = process.env;
+if (!AURORA_API_KEY) {
+  throw new Error("*** AURORA_API_KEY NOT FOUND IN ENV");
+}
+
+const AURORA_URL = "https://mainnet.aurora.dev/" + AURORA_API_KEY;
+const SAFE_SERVICE_URL = "https://safe-transaction.aurora.gnosis.io/";
+const provider = new JsonRpcProvider(AURORA_URL);
+
+if (!SAFE_SIGNER_MNEMONIC) {
+  throw new Error("*** SAFE_SIGNER_MNEMONIC NOT FOUND IN ENV ***");
+}
+
+const deployer = Wallet.fromMnemonic(SAFE_SIGNER_MNEMONIC).connect(provider);
+const allocPoint = 0;
+
+console.info("*** Using deployer address: ", deployer.address);
+console.info("*** Using SAFE_SERVICE_URL: ", SAFE_SERVICE_URL);
+
+async function main() {
+  console.info("*** Proposing updating pool allocation ***");
+
+  let allocationConfig: AllocationConfig | undefined;
+  try {
+    allocationConfig = await fs.readJSON("./allocationConfig.json");
+    console.info("*** allocationConfig.json found ***");
+    console.info(JSON.stringify(allocationConfig));
+  } catch (err) {
+    console.info("*** No allocationConfig.json found");
+  }
+
+  const service = new SafeService(SAFE_SERVICE_URL);
+  const signer = deployer;
+  console.log("Setup SafeEthersSigner");
+  const ethAdapter = new EthersAdapter({ ethers, signer });
+  const safe = await Safe.create({ ethAdapter, safeAddress: ops });
+  const safeSigner = new SafeEthersSigner(safe, service, provider);
+
+  if (allocationConfig) {
+    if (typeof allocationConfig?.Rewarder === "string") {
```

Comment:
Can reduce nesting here
```suggestion
  if (allocationConfig && typeof allocationConfig?.Rewarder === "string") {
```